bbc/simorgh
# Wsteam1 417 brand colours

Resolves JIRA [WSTEAM1-418](https://jira.dev.bbc.co.uk/browse/WSTEAM1-418)

❗❗❗❗❗❗❗ DO NOT MERGE UNTIL: ❗❗❗❗❗❗❗
======
- Delete src/app/components/ThemeProvider/colourSimilarity.py which is a script file written in python created to help us audit the colours. 

Overall changes
======
Replaces Simorgh's Legacy colours with BBC approved Brand colours.

Code changes
======

**_Renamed_**
- POSTBOX --> NEWS_CORE
- SPORT_YELLOW --> SPORT_CORE
- PHILIPPINE_GREY --> GREY_5

**_Replaced With Similar Colour_**
Similarity was calculated using the following [approach](https://www.baeldung.com/cs/compute-similarity-of-colours#improved-approach)
- GHOST (`#FDFDFD`) --> GREY_1 (`#FEFEFE`)
- SPORT_MIST (`#F7F7F5`) --> GREY_2 (`#F6F6F6`)
- LUNAR (`#F2F2F2`) --> GREY_2 (`#F6F6F6`)
- PEBBLE (`#AEAEB5`) --> GREY_4 (`#B0B2B4`)
- RHINO (`#5A5A5A`) --> GREY_6 (`#545658`)
- SHADOW (`#3F3F42`) --> GREY_7 (`#3A3C3E`)
- EBON (`#222222`) --> GREY_8 (`#202224`)
- MIDNIGHT_BLACK (`#121212`) --> GREY_10 (`#141414`)
_Note: Replacement included deletion of the original colour_

**_Replaced But Waiting for UX Approval**
- CONSENT_ACTION (`#F6A21D`) --> ALERT_LIGHT (`#FFCE2B`)
- CONSENT_BACKGROUND (`#323232`) --> GREY_10 (`#141414`)
- CONSENT_CONTENT (`#BEBEBE`) --> GREY_2 (`#F6F6F6`)
- KINGFISHER (`#11708C`) --> LIVE_DARK (`#006666`)
- CLOUD_LIGHT (`#BABABA`) --> GREY_4 (`#B0B2B4`)
- SPORT_SILVER (`#DBDBDB'`) --> GREY_3 (`#E6E8EA`)
_Note: Replacement included deletion of the original colour_

**_Deleted_**
- BLUEJAY (`#0F556C`)
- BLUEJAY_LHT (`#C3DEE7`)
- CHALK (`#ECEAE7`)
- CLOUD_DARK (`#757575`)
- DARK_SALTIRE (`#23104C`)
- DIM_GREY (`#696969`)
- LE_TEAL (`#09838B`)
- LUNAR_LIGHT (`#F8F8F8`)
- NEWSROUND_PURPLE_30 (`#9159A8`)
- OAT_LHT (`#F5F3F1`)
- ORBIT_GREY (`#4C4C4C`)
- POSTBOX_30 (`#EAB3B3`)
- SPORT_YELLOW_30 (`#BB9A31`)
- STORM (`#404040`)
- WEATHER_BLUE (`#067EB3`)

**[_Audit In Progress_](https://paper.dropbox.com/doc/WS-colour-palette--B6sI2oz6TUHu8P01FAOCTCwzAg-ckaoptDrRSZUHArklqpra)**
- METAL (`#6E6E73`)
- STONE (`#D5D0CD`)
- ARCHIVE_BLUE (`#D5D0CD`)
- NEWSROUND_PURPLE (`#6C22D6`)

**More Code Changes**
- Removed unused key `BRAND_FOREGROUND` from service themeing
- Removed unused key `BRAND_HIGHLIGHT` from service themeing
- Removed unused key `BRAND_BORDER` from service themeing

Testing
======
I would suggest to only test Replaced Colours. Components where these colours were used can be found [here](https://paper.dropbox.com/doc/WS-colour-palette--B6sI2oz6TUHu8P01FAOCTCwzAg-ckaoptDrRSZUHArklqpra)
1. Navigate to a component with replaced colour
2. See that the replacement correctly replaced the colour.
3. See that design are the same as original but for the replaced colour.
4. See that the colour does not violate colour contrast.

Helpful Links
======
Colour Audit: https://paper.dropbox.com/doc/WS-colour-palette--B6sI2oz6TUHu8P01FAOCTCwzAg-ckaoptDrRSZUHArklqpra

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
